scout/server/links.py

```diff
@@ -266,6 +267,12 @@ def smart(smart_domain):
     return link.format(smart_domain)
 
 
+def varsome(build, refseq_id, protein_sequence_name):
```

Comment:
It would be nice to handle the situation in which refseq_id and /or protein_sequence_name are missing. In that case the fuction should return None I think.

Another observation: default build passed is 37, which for the same variant as above, will convert the link in something like `https://varsome.com/variant/hg37/NM_015450%3Ac.903G>T`, that in turn is converted into `https://varsome.com/variant/hg19/NM_015450%3Ac.903G>T` by Varsome.

I guess it's OK.

Have you tested the **Varsome link created for variants from cases with genome build 38**? Because I don't think the right build is propagated to this function. It's not your fault of course because the error must have been there from before, but could you try to fix it?

Tip: You could simulate a case with genome build 38 by modifying one of the demo config files, for instance `scout/demo/643595.config.yaml`, writing `human_genome_build: 38`

Reply:
As soon as we merge #1808 this problem will be fixed, see that PR!

Reply:
Problem fixed. You might want to rebase this PR perhaps!

Reply:
I added so the function returns None if missing refseq id or protein sequence name now. I also had to add the genome build as input to the the function  add_gene_info  in the file server/blueprints/variant/utils.py to make the hg38 build work properly.